cli/src/opts/wallet/mod.rs

```diff
@@ -27,70 +28,67 @@ pub use multi_wallet::*;
 
 pub mod error;
 
-/// The wallet options can either be:
-/// 1. Ledger
-/// 2. Trezor
-/// 3. Mnemonic (via file path)
-/// 4. Keystore (via file path)
-/// 5. Private Key (cleartext in CLI)
-/// 6. Private Key (interactively via secure prompt)
-/// 7. AWS KMS
+// The raw wallet options can either be:
+// 1. Private Key (cleartext in CLI)
+// 2. Private Key (interactively via secure prompt)
+// 3. Mnemonic (via file path)
 #[derive(Parser, Debug, Default, Clone, Serialize)]
-#[clap(next_help_heading = "Wallet options", about = None, long_about = None)]
-pub struct Wallet {
-    /// The sender account.
-    #[clap(
-        long,
-        short,
-        help_heading = "Wallet options - raw",
-        value_name = "ADDRESS",
-        env = "ETH_FROM"
-    )]
-    pub from: Option<Address>,
-
+#[clap(next_help_heading = "Wallet options - raw", about = None, long_about = None)]
+pub struct RawWallet {
```

Comment:
Let's add a bit of docs on what this is (essentially a wrapper for the raw data for `Wallet`)

Reply:
Done